packages/cdk-graph-plugin-diagram/src/internal/graphviz/entities/nodes.ts

```diff
@@ -59,6 +59,11 @@ function marshallLabelForRendering(original: string): MarshalledLabel {
   return { original, label, lines: lines.length };
 }
 
+/**
+ * Positional coordinates for a node
+ */
+export type NodePosition = { x: number; y: number };
```

Comment:
Please could we specify what the units are for these? I think maybe it's inches for graphviz?

Reply:
Inches it is indeed